src/SolutionTemplate/UnoSolutionTemplate.net6/Mobile/Mobile.net6.vstemplate

```diff
@@ -0,0 +1,168 @@
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
+  <TemplateData>
+    <Name>UnoQuickStart.macOS</Name>
```

Comment:
why macOS?

Reply:
It's actually never shown to the user, but to avoid future name collisions best is to fix it :)